runbooks/generate-onlyswaps-config.md

```diff
@@ -0,0 +1,89 @@
+# Generating an ONLYswaps configuration file
+
+## Prerequisites
+- you've successfully completed [the ceremony](./run-ceremony-operator.md) runbook
+- you have received a final keyshare.pub file from the ceremony containing your public key (amongst others)
+- you've installed the latest [ADKG-CLI](./installing-cli.md)
+
+## Steps
+1. Identify the paths for all your key material, you will need the following files:
+- your operator private key, generated during [the key generation runbook](./operator-key-generation.md)
+- the group.toml file, shared to you by the leader [before the ceremony](./pre-ceremony-operator.md) 
+- your multiaddr, chosen by you [before the ceremony](./pre-ceremony-operator.md) 
+- your member ID, identified in the [first stages of the ceremony](./run-ceremony-operator.md)
+- the ADKG public output file, created [during the ceremony](./run-ceremony-operator.md)
+- the ADKG private output file, created [during the ceremony](./run-ceremony-operator.md)
+- (optional) the address of the router contract for ONLYswaps. There is a default one embedded in the configuration process, but if it's changed you may need a new one.
+
+2. Run the following command, replacing with the relevant paths from above:
+```bash
+adkg-cli generate-onlyswaps-config \
+  --operator-private /path/to/private \
+  --group /path/to/group.toml \
+  --adkg-public /path/to/public \
+  --adkg-private /path/to/share \
+  --multiaddr "/dns/example.org/tcp/1234" \
```

Comment:
we shouldn't advertise dns here, we can probably use `/ip4/0.0.0.0/tcp/1234` as an example